lib/src/git_backend.rs

```diff
@@ -1748,7 +1770,12 @@ mod tests {
             .find_commit(Oid::from_bytes(read_commit_id.as_bytes()).unwrap())
             .unwrap();
         let git_tree = git_repo.find_tree(git_commit.tree_id()).unwrap();
-        assert!(git_tree.iter().all(|entry| entry.filemode() == 0o040000));
+        // All but the README are trees
+        assert!(git_tree
+            .iter()
+            .rev()
+            .skip(1)
```

Comment:
It would seem nicer to filter by whether `entry.name() == Some("README")`, but I could have missed some reason it wouldn't work.

Reply:
Yeah, that's much better. Done.